sdk/sdk-api/src/main/java/mil/emp3/api/Point.java

```diff
@@ -90,11 +92,13 @@ public IGeoIconStyle getIconStyle() {
      */
     @Override
     public void setIconURI(final String sURL) {
-        if(android.webkit.URLUtil.isValidUrl(sURL)) {
-            ((IGeoPoint) this.getRenderable()).setIconURI(sURL);
-        } else {
-            throw new IllegalArgumentException("Invalid Input, " + sURL + " is not a valid URL");
+        if(!android.webkit.URLUtil.isValidUrl(sURL)) {
+            final File testLocation = new File(sURL.substring(5));
```

Comment:
I know why you are removing the first 5 characters from the string, but you should add a detailed comment and check the length of the string as well. Also your branch is out of date please update